app/controllers/planning_applications_controller.rb

```diff
@@ -127,8 +128,6 @@ def validate_documents
     end
   end
 
-private
```

Comment:
Why did we un-`private` these methods?

Reply:
..entirely by accident 😩 Thank you very much for picking up on this!